lib/SILOptimizer/Mandatory/ClosureLifetimeFixup.cpp

```diff
@@ -725,6 +725,9 @@ static SILValue tryRewriteToPartialApplyStack(
   
   SmallSetVector<SILValue, 4> borrowedOriginals;
   
+  unsigned appliedArgStartIdx =
```

Comment:
I wouldn't fix this in this commit, but I think this is automatically fixed for you if you use callee conventions.

Reply:
Sorry, the API is actually on ApplySite.

Reply:
It's somewhere around here: https://github.com/apple/swift/blob/main/include/swift/SIL/ApplySite.h#L341. If there isn't an exact one for what you need, I would add it here. We do not want people to muck around with indices for partial apply since it is pretty easy to mess up (as you see here)